test/firestore_test.dart

```diff
@@ -872,6 +872,15 @@ void main() {
           snapshot.docs[0].data()["text"], anyOf("hello", "hi", "ahoj", "cau"));
     });
 
+    test("collectionGroup", () async {
+      var group = firestore.collectionGroup("index");
+      var snapshot = await group.get();
+
+      expect(snapshot.size, 4);
```

Comment:
This will fail with the current test setup, but I'll fix it up after I land it.

Reply:
What causes it to fail? I mentioned needing v2 rules, but in my testing I found they're only needed to query groups, not to get them, and it passes on my local set up.

Reply:
My test account has a lot of old groups that get counted. See my fix. Pretty trivial.

Reply:
Ah, gotcha. 👍